code/modules/mob/living/simple_animal/hostile/megafauna/king_of_goats.dm

```diff
@@ -78,13 +78,16 @@ Difficulty: Insanely Hard
 	icon_state = "king_goat2"
 	icon_living = "king_goat2"
 	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab = 4, /obj/item/clothing/head/goatpelt/king = 1)
+	achievement_type = /datum/award/achievement/boss/king_goat_kill
+	crusher_achievement_type = /datum/award/achievement/boss/king_goat_crusher
 	health = 750
 	maxHealth = 750
 	armour_penetration = 50
 	melee_damage_lower = 40
 	melee_damage_upper = 60
 	environment_smash = ENVIRONMENT_SMASH_RWALLS
 	pixel_y = 5
+	crusher_loot = list(/obj/item/crusher_trophy/king_goat)
 
 	var/spellscast = 0
```

Comment:
move stun_chance to above the `var/`s

Reply:
these should also be doc'd

Reply:
added docs but moving stun_chance above var/spellscast breaks it since phase 1 king goat uses stun_chance unless I misread what you meant

Reply:
the var positioning shouldn't matter in the definition, it's just for consistency (`var/` goes below pre-defined variables).